### PR TITLE
[Student Learning] [SL-455] [FIXED] Refactor PaneButton to not be a clickable div

### PR DESCRIPTION
**Pull request by levadadenys**
### [Student Learning] [SL-455] Refactor PaneButton to not be a clickable div

No visual changes intended, just refactor the PaneButton and not break anything

This is a fixed version of previos [PR](https://github.com/code-dot-org/code-dot-org/pull/50413)

Before:
Знімок екрана 2023-02-23 о 13 05 25
Знімок екрана 2023-02-23 о 13 27 15

After:
Знімок екрана 2023-02-23 о 13 05 18
Знімок екрана 2023-02-23 о 13 30 10

## Links

- jira ticket: [SL-455](https://codedotorg.atlassian.net/browse/SL-455?atlOrigin=eyJpIjoiMWYxZDJkM2ExODBlNDE5OTg3ZmMxMjNjYmNiZWYwYWUiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/common-styles.module.scss [hunk 1]
```diff
@@ -20,6 +20,12 @@
   &Unfocused {
     background-color: color.$lighter_purple;
     color: color.$dark_charcoal;
+
+    button {
+      span {
+        color: color.$dark_charcoal;
+      }
+    }
```
Comment (sanchitmalhotra126) [inline apps/src/common-styles.module.scss:28]: Just confirming, do we want to apply this to all `button span`s, not just `PaneButton`? also should we copy this into commonStyles.js (based on the comment at the top of the file)?

File: apps/src/templates/pane-header.module.scss [hunk 2]
```diff
@@ -33,9 +51,18 @@
     background-color: #606060;
     color: #bfbfbf;
     border: none;
+
+    span {
+      color: #bfbfbf;
```
Comment (sanchitmalhotra126) [inline apps/src/templates/pane-header.module.scss:56]: Is there a color constant we can use for this?

File: apps/src/common-styles.module.scss (hunk 1 above)
Comment (levadadenys) [reply apps/src/common-styles.module.scss:28]: We're applying this to all the buttons that are inside of purpleHeader class right now. And the only buttons inside that purpleHeader is paneButtons
image

As far as I remember I've asked about commonStyles.js and the answer was not to use it and get rid of it where possible

File: apps/src/templates/pane-header.module.scss (hunk 2 above)
Comment (levadadenys) [reply apps/src/templates/pane-header.module.scss:56]: It's a specific color that we're only using in this exact file, so there're no constant for it

File: apps/src/common-styles.module.scss (hunk 1 above)
Comment (sanchitmalhotra126) [reply apps/src/common-styles.module.scss:28]: Oh got it, missed that it was nested inside .purpleHeader, thanks for clarifying!

**Review by sanchitmalhotra126 — APPROVED** [review]

**Pull request merged**